diff --git a/packages/utils/request/index.js b/packages/utils/request/index.js
index f3909e0..b95d365 100644
--- a/packages/utils/request/index.js
+++ b/packages/utils/request/index.js
@@ -11,7 +11,7 @@ type ResPayload = Object & {
   timeout: boolean,
 };
 
-type PromiseRes = (payload: ResPayload) => void;
+// type PromiseRes = (payload: ResPayload) => void;
 
 type Cancelled = Object & {
   cancelled: boolean,
