diff --git a/packages/components/Card/index.js b/packages/components/Card/index.js
index b0df5df..f273c36 100644
--- a/packages/components/Card/index.js
+++ b/packages/components/Card/index.js
@@ -43,7 +43,7 @@ type Props = {
    * - sets special icon and border style
    * - opts: 'selected' | 'selectable' | 'deselected' | 'warning' | 'error'
    */
-  status?: ?CardStatusType,
+  status?: string,
   /** asset url */
   backgroundImage?: string,
   backgroundSize?: string,
@@ -133,7 +133,7 @@ const wrapperStyles = ({
   ${clickable &&
     `
   cursor: pointer;
-  
+
     &:hover {
       box-shadow: 0 0 ${boxShadow.blur} ${boxShadow.color};
       transform: scale(1.03);
@@ -221,7 +221,7 @@ const wrapperStyles = ({
     background-repeat: repeat;
     background-image: url('${backgroundImage}');
   `}
-  
+
   ${backgroundSize &&
     css`
       background-size: ${backgroundSize};
